### PR TITLE
added jammy image

### PR DESCRIPTION
**Pull request by WRFitch**
## Release Notes
- Update to use new jammy container. 

## Context
- This PR isn't being merged until the tests are updated in the bundle repo, since they're currently failing and I don't want to make the fix more complicated. 

## Testing
Integration tests are currently running.

### CODE REVIEW

**Review by taurus-forever — COMMENTED** [review]
LGTM if new pgbouncer version works for us.
Up to Marcelo to decide!

File: .github/workflows/release.yaml
Comment (marceloneppel) [inline .github/workflows/release.yaml:47]: Is [this](https://github.com/canonical/pgbouncer-k8s-operator/actions/runs/3846492286/jobs/6553180944#step:4:23) still a problem?

**Review by marceloneppel — APPROVED** [review]
LGTM! Just take a look on the issue with the image when publishing to Charmhub.

File: .github/workflows/release.yaml
Comment (WRFitch) [reply .github/workflows/release.yaml:47]: I think overriding pgbouncer-image is unnecessary, since we can just pull it from dockerhub

**Review by taurus-forever — DISMISSED** [review]
LGTM, please double check Marcelo comment about `resource-overrides: "pgbouncer-image:3"` (after merging).

**Review by marceloneppel — APPROVED** [review]

**Review by taurus-forever — COMMENTED** [review]
After merging review: LGTM! Tnx!

**Pull request merged**
